wazo_acceptance/helpers/webrtc_phone.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2015-2025 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import docker
+from playwright.sync_api import Browser, Page
+from wazo_test_helpers import until
+
+from wazo_acceptance.helpers.phone import Phone
+
+CONTAINER_NAME = 'acceptance-webrtc-test-helper'
+EXPOSED_PORT = 3000
+
+
+class WebRTCPhone(Phone):
+
+    def __init__(self, context, browser: Browser, page: Page, sipUsername: str, sipPassword: str):
+        self.page = page
+        self._browser = browser
+        self._client = docker.from_env()
+        self.sipUsername = sipUsername
```

Comment:
```suggestion
        self.sip_username = sip_username
```
Same here, camelCase -> snake_case